### ISSUE TITLE
[gateway] Global logging config

### ISSUE THREAD

**Issue by LuisTAC** [gravitee-io/issues#1177]
Instead of configuring logging in each API, it would be nice to have the logging mode applied to all APIs in a gateway.

### Expected Behavior

It would be useful to configure logging on the whole gateway, instead of each API.

### Current Behavior

Currently, the only way to enable logging is per API (on it's proxy configuration), or per API path (adding a [Logging policy](https://github.com/gravitee-io/gravitee-policy-logging)).

### Possible Solution
A possible solution, that I believe doesn't compromise current behavior of APIs, is to add another property to the `gravitee.yml` configuration file (for example `defaultLoggingMode`, assuming `NONE`, ..., `CLIENT_PROXY` values) that indicates the default logging mode when creating a new API. This way, upon adding a new API, it will already be with the defined mode. If, for some reason, a specific API isn't supposed to have logs enabled, its logging mode can be modified before starting and 'publishing' it. In order not to interfere with the current behavior, the default value of this `defaultLoggingMode` property should be `NONE`
